### PR TITLE
Availabilities

### PR DESCRIPTION
**Pull request by Wennxn**
#64

### CODE REVIEW

File: components/drawer/DrawerContent.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ const DrawerContent = (props) => {
         Actions.auth();
         await AsyncStorage.removeItem("token");
         removePushToken(props.auth.userId);
+        if (props.naloxoneAvailability)
```
Comment (Asiapenguin) [inline components/drawer/DrawerContent.js]: This should also set the online status to false.

File: screens/LoginScreen.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ class LoginScreen extends Component {
     console.log("[DEBUG] username is " + username + ", password is " + password);
     getDeviceLocation((coords) => { this.props.setLocalLocation({ coords }) })
     this.props.setLoading(true);
-    this.props.loginHandler({ username: username, password: password }, rememberMe);
+    this.props.loginHandler({ username: username, password: password, naloxoneAvailability: this.props.naloxoneAvailability }, rememberMe);
```
Comment (Asiapenguin) [inline screens/LoginScreen.js]: Why do we need the naloxone boolean here?

File: store/actions/AuthActions.js [hunk 3]
```diff
@@ -53,6 +54,7 @@ export const loginHandler = (credential, rememberMe) => {
       .then(async (response) => {
         dispatch(setLoading(false));
         dispatch(login(response.data, rememberMe));
+        dispatch(setStatus(response.data.id, response.data.token, { "naloxoneAvailability": response.data.naloxoneAvailability }))
```
Comment (Asiapenguin) [inline store/actions/AuthActions.js]: Is this supposed to be setting the online status instead?

Comment (Wennxn) [reply store/actions/AuthActions.js]: Online status is already set by the backend through the endpoint

Comment (Asiapenguin) [reply store/actions/AuthActions.js]: Oh i see, but why do we need to update the naloxone at login? The user hasnt touched the switch yet.

Comment (Asiapenguin) [reply store/actions/AuthActions.js]: oh is it because its from the server and you are adding it to redux state lmao im stupid

Comment (Wennxn) [reply store/actions/AuthActions.js]: I was about to say that, glad you figured~

**Pull request merged**